Add Policy Packs for Azure from ext sheet. (#777)
turbot/guardrails-samples: policy_packs/azure/storage/enforce_public_access_level_for_containers/README.md

---
categories: ["security"]
---

> # Enforce Public Access Level for Azure Storage Containers

Reviewer: ```suggestion
# Enforce Block Public Access for Azure Storage Containers
```

Reviewer: Please update the policy pack name accordingly.

Enforcing Public Access Level for Azure Storage Containers ensures that data within the containers is either publicly accessible or restricted as per the set access levels. This security measure helps prevent unauthorized access and data breaches by managing who can access the container's data over the internet.

> This [policy pack](https://turbot.com/guardrails/docs/concepts/resources/smart-folders) can help you configure the following settings for storage accounts:

Reviewer: ```suggestion
This [policy pack](https://turbot.com/guardrails/docs/concepts/resources/smart-folders) can help you configure the following settings for containers:
```

> - Enforce public access level is set to Private

Reviewer: ```suggestion
- Set public access level to Private (no anonymous access)
```
